app/views/embeddable/multiple_choices/_form.html.haml

```diff
@@ -39,6 +56,34 @@
     :javascript
       $(function() {
           $('.wysiwyg').wysiwyg({"controls": { "html": { "visible": true }, "h1": { "visible": false }, "h2": { "visible": false }, "h3": { "visible": false } } });
+
+          var vert = $('##{vertical_id}'),
+              horiz = $('##{horizontal_id}'),
+              likert = $('##{likert_id}'),
+              show_as_menu = $('##{show_as_menu_id}');
+
+          show_as_menu.click(function() {
+            if($(this).is(':checked')){
+              vert.attr('disabled',true);
+              horiz.attr('disabled',true);
+              likert.attr('disabled',true);
+            }else{
+              vert.removeAttr('disabled');
+              horiz.removeAttr('disabled');
+              likert.removeAttr('disabled');
+            }
+          });
+
+          if(show_as_menu.is(':checked')){
+            vert.attr('disabled',true);
+            horiz.attr('disabled',true);
+            likert.attr('disabled',true);
+          }else{
+            vert.removeAttr('disabled');
+            horiz.removeAttr('disabled');
+            likert.removeAttr('disabled');
```

Comment:
If you add an id to `field_set_tag 'Layout'` then you can use a jquery selector to disable or enable all of the radio_buttons inside of this field_set with one call.  So if the id was a variable `embeddabe_layout_options` you could do $radios = $('##{embeddable_layout_options} input')
And then $radios.attr('disabled', true)

You should also clean up the duplicated code here.